### PR TITLE
update manifests: secure tiller/mongo, isolate nginx

### PR DESCRIPTION
**Pull request by prydonius**
- updates manifests to the latest from kubeapps/manifest
- moves nginx-ingress-controller to the kubeapps namespace for isolation
- secures tiller and mongodb
- removes the mongodb-password secret as only mongodb-root-password is
  used

### CODE REVIEW

**Comment by prydonius** [comment]
@ngtuna this is going to clear the `created-by` label, so we should probably wait for that to be added to the manifests and re-sync

File: cmd/up.go [hunk 1]
```diff
@@ -29,22 +29,22 @@ import (
 	"github.com/ksonnet/kubecfg/utils"
 	"github.com/kubeapps/kubeapps/pkg/gke"
 	"github.com/spf13/cobra"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
```
Comment (prydonius) [inline cmd/up.go]: the work of goimports

File: cmd/up.go [hunk 2]
```diff
@@ -29,22 +29,22 @@ import (
...
-	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 const (
-	GcTag      = "bitnami/kubeapps"
-	KubeappsNS = "kubeapps"
-	KubelessNS = "kubeless"
-	SystemNS   = "kube-system"
-	Kubeapps_NS = "kubeapps"
+	GcTag          = "bitnami/kubeapps"
```
Comment (prydonius) [inline cmd/up.go]: gofmt

**Comment by ngtuna** [comment]
Totally agree. I made this PR to add the label to the manifest https://github.com/kubeapps/manifest/pull/26

**Comment by prydonius** [comment]
Okay created-by labels are back as per https://github.com/kubeapps/manifest/pull/26

**Comment by ngtuna** [comment]
lgtm. I tested it on my setup. it worked.

**Pull request merged**